src/components/Tooltip/style.css

```diff
@@ -0,0 +1,227 @@
+.wrapper {
+  position: relative;
+  display: inline-block;
+}
+
+.original {
+  position: relative;
+}
+
+.Tooltip {
+  backface-visibility: hidden;
+  border-radius: var(--rvr-tooltip-border-radius);
+  box-shadow: var(--rvr-tooltip-box-shadow);
+  box-sizing: border-box;
+  font-size: 14px;
+  position: absolute;
+
+  /* transform: translate3d(0, 0, 0); */
+  transition: opacity 0.3s ease, transform 0.3s ease;
+  will-change: opacity, transform;
+  max-width: 100vw;
+  z-index: var(--rvr-tooltip-base-z-index);
+  opacity: 0;
+  visibility: hidden;
+  pointer-events: none;
+
+  /* original code */
+
+  /* position: absolute;
+  visibility: hidden;
+  opacity: 0;
+  padding: 5px 10px;
+  min-width: 88px;
+  white-space: nowrap;
+  color: var(--rvr-grey-80);
+  */
+}
+
+.Tooltip::before,
+.Tooltip::after {
+  pointer-events: none;
+  border: solid transparent;
+  content: '';
+  height: 0;
+  width: 0;
+  position: absolute;
+}
+
+.open {
+  opacity: 1;
+  visibility: visible;
+  pointer-events: all;
+}
+
+.tooltipInnerWrapper {
+  background: var(--rvr-white);
+  border-radius: var(--rvr-tooltip-border-radius);
+  padding: 8px;
+  position: relative;
+  box-sizing: border-box;
+  height: 100%;
+  width: 100%;
+}
+
+.textContent {
+  white-space: nowrap;
+  word-wrap: none;
+  text-align: center;
+}
+
+.tooltipClose {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  color: var(--rvr-gray-lite-2);
+  margin: 0;
+  margin: 5px;
+  padding: 0;
+  position: absolute;
+  right: 5px;
+  top: 5px;
+  z-index: 51;
```

Comment:
I'm wondering why this is 51, does it have any special meaning?

Reply:
This was just to make sure that the tooltip content, the white arrow, and the arrow shadow had correct z-index relative to each other.

This is a bit of a magical number. I should put it with the other tooltip CSS custom properties